next.config.mjs

```diff
@@ -61,6 +68,11 @@ const nextConfig = {
 const sentryWebpackPluginOptions = {
   dryRun: !process.env.SENTRY_AUTH_TOKEN,
   silent: true,
+  deploy: {
+    env: process.env.SENTRY_ENVIRONMENT,
+  },
 };
 
-export default withSentryConfig(nextConfig, sentryWebpackPluginOptions);
+export default process.env.SENTRY_DSN
+  ? withSentryConfig(nextConfig, sentryWebpackPluginOptions)
+  : nextConfig;
```

Comment:
Prevents Netlify build from crashing if Sentry DSN is not defined. Otherwise, we see this error:

```
Error [SentryError]: Invalid Dsn
  at new SentryError (/opt/build/repo/node_modules/@sentry/utils/dist/error.js:9:28)
  at Dsn._fromString (/opt/build/repo/node_modules/@sentry/utils/dist/dsn.js:39:19)
```